### PR TITLE
fix javadoc.

### PR DESCRIPTION
**Pull request by RobAltena**
Fix 4 review items from #7997

Only javadoc fixes, no code changes.

### CODE REVIEW

File: nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/factory/Nd4j.java [hunk 1]
```diff
@@ -563,7 +563,7 @@ public static INDArray create(int[] sliceShape, float[]... arrays) {
     }
 
     /**
-     * @see #create(LongShapeDescriptor)
+     * @see #create(LongShapeDescriptor, boolean) with initialize set to true.
```
Comment (AlexDBlack) [inline nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/factory/Nd4j.java]: Use `See {@link #...} with ...` instead of `@see`

Why? look how this renders:
image

It's a quirk of JavaDoc that it'll render the method name only if the `@see` doesn't contain any text. If it does include any text, the method name won't be included. So even though it reads perfectly fine in the source, it won't read properly when rendered to HTML.

Or:
https://github.com/eclipse/deeplearning4j/blob/57efc2bf956af1140c00675aca7c8bd8a67f29bf/deeplearning4j/deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/multilayer/MultiLayerNetwork.java#L3706
(note the seemingly odd `@see ModelSerializer ModelSerializer ...` - first one is the link, second one is part of the message)
vs. 
https://deeplearning4j.org/api/latest/org/deeplearning4j/nn/multilayer/MultiLayerNetwork.html
image


In other words, this will render as `See Also: with initialize set to true.` which isn't good/readable.

image

**Comment by RobAltena** [comment]
Approved per DM.
